packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableSelector.ts

```diff
@@ -20,7 +25,7 @@ export default function createTableSelector(
         elementData: CreateElementData,
         helperType: 'CellResizer' | 'TableInserter' | 'TableResizer' | 'TableSelector'
     ) => void,
-    contentDiv?: EventTarget | null
+    contentDiv?: HTMLElement
```

Comment:
In line 32, it checks if table is visible to editor using this contentDiv. However, now contentDiv is changed to be the "container", which can be different than before, so isTableTopVisible() may return different result.

I think we should not change this contentDiv, but add one more parameter for the container.

Reply:
Specific to this case, the top value of the container and contentDiv should be the same, and if they are not what happens is that it renders a TableSelector square that is not visible. It is the same that happens currently with TableResizer and the bottom.

Reply:
So that added an additional requirement that the container needs to have the same top position with editor.

We can actually remove this limitation by adding another parameter, and try keep using position:fix to simplify t he calculation of position.

position:fixed means the position is related to the whole window, no matter who the parent is, which is a very convenient style to use, especially when there is complex scrolling, zooming, margin, ...